src/main/zapHomeFiles/hud/display.js

```diff
@@ -617,7 +617,7 @@ Vue.component('websocket-message-modal', {
 
 		eventBus.$on('showWebSocketMessageModal', data => {
 			const date = new Date(Number(data.msg.timestamp));
-			self.time = date.getHours() + ':' + date.getMinutes() + ':' + date.getSeconds() + '.' + date.getMilliseconds();
+			self.time = date.toJSON().slice(11, 23);
```

Comment:
Use `toTimeString()` which makes it more clear what the intent is. Worth extracting a function into `Utils` to stop duplicating in each place.